File: server/Data/Entities/Game.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace Api.Data.Entities
{
  public class Game
  {
    [Key]
    public int Id { get; set; }
    public string HostId { get; set; }
    public GameStatus Status { get; set; }
    public DateTime CreationDate { get; set; }
    public Difficulty Difficulty { get; set; }
    public TimeSpan CardSpeed { get; set; }
    public int RoundsToWin { get; set; }

    public ICollection<Player> Players { get; } = new List<Player>();
  }
}

File: client/src/types/Game.ts
import { Player } from './Player'
import { Difficulty } from './Difficulty'
import { GameStatus } from './GameStatus'

export interface Game {
  id: number
  hostId: string
  status: GameStatus
  creationDate: string
  difficulty: Difficulty
  cardSpeed: number
  roundsToWin: number
  players: Array<Player>
}
